src/features/canvass/l10n/messageIds.ts

```diff
@@ -17,6 +17,12 @@ export default makeMessages('feat.canvass', {
       numberOfFloorsInput: m('Number of floors'),
       numberOfHouseholdsInput: m('Households per floor'),
     },
+    delete: {
+      title: m('Confirm deletion of household'),
```

Comment:
This passive voice sounds a bit weird to me. Is this inspired by a similar dialog elsewhere in the code? What could be some other options?

Reply:
I've change it for a more active voice "Delete household"